api/conf/conf.yaml

```diff
@@ -17,10 +17,15 @@
 
 conf:
   listen:
-    host: 127.0.0.1     # `manager api` listening ip or host name
```

Comment:
We still support configuring the host. So can we comment on this instead of removing them?

Reply:
@starsz your way is better

Reply:
updated.